src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    line-height: 60px;
+    text-decoration: none;
+    margin-left: 20px;
+    text-transform: uppercase;
+  }
+
+  &__item:first-child {
+    margin: 0;
+  }
+
+  &__link {
+    height: 60px;
+    text-decoration: none;
+    color: $main-text;
+    display: block;
+    transition-duration: $trans-duration;
+  }
+
+  &__link:hover {
+    color: $accent_colour;
+    cursor: pointer;
+  }
```

Comment:
Same here with transition. also cursor: pointer; should be always, not only on hover - move to `&__link`